cmake/FindHwloc.cmake

```diff
@@ -134,7 +134,30 @@ if(WIN32)
 
 else()
 
-  # Find with pkgconfig
+  if(CMAKE_CROSSCOMPILING)
+
+  find_path(Hwloc_INCLUDE_DIRS
+    NAMES
+      hwloc.h
+    PATHS
+      ENV HWLOC_ROOT
+  )
+
+  find_library(Hwloc_LIBRARIES
+    NAMES
+      hwloc
+    PATHS
+      ENV HWLOC_ROOT
+  )
+
+  if(Hwloc_INCLUDE_DIRS AND Hwloc_LIBRARIES)
+    # this is kindof arbitrary
```

Comment:
Yess. I think it's possible to get pkg-config to get the information from target (not host) files, but have to look into it more.